packages/validate/src/lib/dom/index.js

```diff
@@ -95,6 +96,26 @@ export const renderErrors = state => {
     });
 };
 
+
+/**
+ * Looks for any value tokens and replaces them within the error message
+ * 
+ * @param state [Object, validation state]
+ * @param groupName [String, validation group] 
+ * 
+ */
+ export const updateMessageValues = (state, groupName) => {
+    let msg = state.groups[groupName].errorMessages[0];
+
+    let values = state.groups[groupName].fields.reduce((acc, field, index, array) => {
```

Comment:
@sarah-storm What is "acc" please? I see how it is being used, but the name is causing me some confusion 👴

Reply:
Hi Nick!  It stands for 'accumulator' which in the reduce function is the variable that [keeps being added to as the function runs over the array](https://www.digitalocean.com/community/tutorials/js-finally-understand-reduce)  .

I could definitely rename it here to represent the error message, but I had seen the variable name used like this in other reducers within the components so had reused.  I will update!